src/redux/book/books.js

```diff
@@ -1,36 +1,27 @@
+/* eslint-disable no-console */
 /* eslint-disable no-undef */
+// import { objectTraps } from 'immer/dist/internal';
```

Comment:
You did great job, however, there is a a tiny issue.
To make your code professional, kindly remove `commented` codes from your file 😊